pages/index.html

```diff
@@ -44,107 +44,93 @@
         </script>
     </head>
 
-    <body id="search" class="CoveoSearchInterface" data-enable-history="true" style="padding: 1em;">
```

Comment:
Why is the padding gone?

Reply:
You didn't answer...

Reply:
I gave you a general answer to your question about your file in a previous comment:
This file is only used for testing purposes. I based my template on the one from the JSUI and cleaned it because there was a main-section in another main section + duplicate code and all.

I don't think there's real value to be gained to further modify it because if you need it for development you're bound to customize it further.

Also my previous comment apply + the one of Analytic:
> As I said, it's based on the Search page template of the coveo/search-ui.
https://github.com/coveo/search-ui/blob/master/pages/All.html I invite to check it out, you won't find an Analytics.